internal/command/user_grant.go

```diff
@@ -144,27 +177,33 @@ func (c *Commands) removeRoleFromUserGrant(ctx context.Context, userGrantID stri
 	return usergrant.NewUserGrantChangedEvent(ctx, userGrantAgg, existingUserGrant.UserID, existingUserGrant.RoleKeys), nil
 }
 
-func (c *Commands) DeactivateUserGrant(ctx context.Context, grantID, resourceOwner string) (objectDetails *domain.ObjectDetails, err error) {
-	if grantID == "" || resourceOwner == "" {
+func (c *Commands) DeactivateUserGrant(ctx context.Context, grantID string, resourceOwner *string, ignoreNotActive bool, check UserGrantPermissionCheck) (objectDetails *domain.ObjectDetails, err error) {
```

Comment:
Why is the resourceowner changed to a pointer if there is no difference if nil or empty?

Reply:
Because this makes it more explicit that it is valid to not pass a resource owner.
Just code style.

Reply:
Valid point, I can revert that

Reply:
done https://github.com/zitadel/zitadel/pull/9978/commits/f93020ff1e0224fc33e1a0dcdab0de66ed615a8a